### PR TITLE
clean up whole documentation

### PR DESCRIPTION
**Pull request by pgerber**
• fix spelling, grammar and clarity issues
• merge dump_database, restore_database and copy_database docs
• add Solr and Nginx glossary entries
• use ``…`` in more instances to improve readability
• consistently use Nice (stop using Nice2)
• move Show Long Running Queries into Show all Queries section
• add some content to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# devops_docs
-Tocco DevOps Documentation
+# Tocco DevOps Documentation
+
+[![Documentation Status](https://readthedocs.org/projects/tocco-devops-docs/badge/?version=latest)](https://tocco-devops-docs.readthedocs.io/en/latest/?badge=latest)
+
+This documentation is based [Sphinx](http://www.sphinx-doc.org/en/stable/) and
```
Comment (ottrob) [inline README.md]: `based on Sphinx`?

File: continuous_delivery/deploy_and_basics.rst [hunk 2]
```diff
@@ -92,7 +92,8 @@ This is only used for production systems, it allows you to deploy a docker image
 system. By default the ``test`` tag is deployed which installs the very same image the test system is using. You can
 however also install an older image formerly installed on the test system.
 
-#. Find the tag name for the version you want to have installed. I looks like this ``test-2017-05-09T12_54_27``.
+#. Find the tag name for the version you want to have installed. I looks like this ``ecap-2017-05-09T12_54_27``.
```
Comment (ottrob) [inline continuous_delivery/deploy_and_basics.rst]: `It looks like` instead of `I looks like...`

File: databases/sql_snippets_postgres.rst [hunk 3]
```diff
@@ -4,52 +4,39 @@ SQL Snippets (Postgres)
 Show Queries Running in Postgres
 --------------------------------
 
-show all queries:
+Show all Queries
```
Comment (ottrob) [inline databases/sql_snippets_postgres.rst]: Im not sure how it's done in general but but lots of words in titles are written uppercase even if they normally would be written lowercase (Queries L4&L7, Running L4).

File: continuous_delivery/deploy_and_basics.rst (hunk 2 above)
Comment (pgerber) [inline continuous_delivery/deploy_and_basics.rst]: agreed

File: databases/sql_snippets_postgres.rst (hunk 3 above)
Comment (pgerber) [inline databases/sql_snippets_postgres.rst]: No one really knows, there isn't an authority that comes up with official rules for English (unlike other languages like German). As general rule, first and last words are capitalized and verbs, nouns, pronouns, adverbs and adjectives.

**Pull request merged**